deploy/utils.sh

```diff
@@ -388,3 +389,57 @@ verify_secret_value_in_pod() {
   $cli_with_timeout "exec -n $APP_NAMESPACE_NAME ${pod_name} -- \
     printenv | grep $environment_variable_name | cut -d '=' -f 2- | grep $expected_value"
 }
+
+get_app_logs_container() {
+    echo "Get logs from the Secrets Provider container"
+    set_namespace "$APP_NAMESPACE_NAME"
+    helm=$(helm ls -aq)
+    $cli_without_timeout get pods
+
+    if [[ -z "$helm" ]]; then
+      pod_name=$($cli_without_timeout get pods --selector app=test-env --no-headers  | awk '{print $1}' )
+      echo "pod_name="$pod_name
+
+      if [[ $pod_name != "" ]]; then
+        $cli_without_timeout logs $pod_name -c cyberark-secrets-provider-for-k8s > "output/$SUMMON_ENV-secrets-provider-logs.txt"
+      fi
+    else
+      pod_name=$($cli_without_timeout get pods --selector app=test-helm --no-headers | awk '{print $1}' )
+      echo "pod_name="$pod_name
+
+      if [[ $pod_name != "" ]]; then
+         $cli_without_timeout logs $pod_name  > "output/$SUMMON_ENV-secrets-provider-logs-with-helm.txt"
+      fi
+    fi
+}
+
+get_conjur_logs_container() {
+    echo "Get logs from Conjur"
+    set_namespace "$CONJUR_NAMESPACE_NAME"
+    selector="role=follower"
+
+    if [ "$CONJUR_DEPLOYMENT" = "oss" ]; then
+      selector="app=conjur-oss"
+    fi
+
+    $cli_without_timeout get pods
+    $cli_with_timeout get pods --selector=$selector --namespace $CONJUR_NAMESPACE_NAME --no-headers | awk '{ print $1 }' | head -1
```

Comment:
you set the namespace and still use the --namespace $CONJUR... so i think this should apply everywhere even though you set the namespace

Reply:
ohh... this is my mistake . I will remove it from here as well. Thanks